Actions/Sign/Sign.psm1

```diff
@@ -1,44 +1,99 @@
-function GetNavSipFromArtifacts
-(
-    [string] $NavSipDestination
-)
-{
-    $artifactTempFolder = Join-Path $([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName())
+<#
+    .SYNOPSIS
+    Installs the dotnet signing tool.
+    .DESCRIPTION
+    Installs the dotnet signing tool if isn't already installed.
+    If you want to avoid installing it on-demand while signing you can run ´dotnet tool install sign --version $Version --global´ to install it on your machine
+    .PARAMETER Version
+    The version of the signing tool to install.
+#>
+function Install-SignTool() {
+        param(
+            [Parameter(Mandatory = $false)]
+            [string] $Version
+        )
 
-    try {
-        Download-Artifacts -artifactUrl (Get-BCArtifactUrl -type Sandbox -country core) -basePath $artifactTempFolder | Out-Null
-        Write-Host "Downloaded artifacts to $artifactTempFolder"
-        $navsip = Get-ChildItem -Path $artifactTempFolder -Filter "navsip.dll" -Recurse
-        Write-Host "Found navsip at $($navsip.FullName)"
-        Copy-Item -Path $navsip.FullName -Destination $NavSipDestination -Force | Out-Null
-        Write-Host "Copied navsip to $NavSipDestination"
-    }
-    finally {
-        Remove-Item -Path $artifactTempFolder -Recurse -Force
-    }
+        if (Get-Command -Name "sign" -ErrorAction SilentlyContinue) {
+            Write-Host "Signing tool is already installed"
+        } else {
+            Write-Host "Signing tool not found. Installing version $Version locally."
+            dotnet tool install sign --version $Version --tool-path SigningTool
```

Comment:
This will install the tool in the path SigningTool under the current path - but what is the current path?
I would think the safest method would be to install the signingtool to a temp folder (random name) and return the sign.exe path and then invoke that - avoiding alias' altogether.